config/routes.rb

```diff
@@ -704,6 +704,7 @@
   # site status API
   get '/status/' => 'status#index', defaults: { format: 'json' }
   get '/website_status/' => 'public#website_status'
+  get '/stats/' => 'stats#index', defaults: { format: 'json' }
```

Comment:
`/website_stats`?

Reply:
Nah, I hate that name.

Note the API routes are: `status` and `stats`, it's only the rails view route that is `website_status`.

Also: in the future there will be `/stats` endpoints added onto `projects`, `sites`, and `regions`. I think it is more consistent in the long run to use a shorter name.